src/api/utils.ts

```diff
@@ -102,7 +102,7 @@ export const toMarket = async (
     const yesPreference =
       Number(marketData.auctionRunningYesPreference ?? 1) /
       Number(marketData.auctionRunningQuantity ?? 1);
-    yesPrice = divideDown(yesPreference);
+    yesPrice = roundToTwo(divideDown(yesPreference));
```

Comment:
theres a function that combines these, used one line below

Reply:
I do not need `roundTwoAndTokenDown` here. it has different logic.

Reply:
i misread